File: Back/src/main/java/com/coomeva/superantojitos/dto/VentaDTO.java
package com.coomeva.superantojitos.dto;

import java.util.Date;

import com.coomeva.superantojitos.domain.TestCliente;

import lombok.Data;

@Data
public class VentaDTO {

	private LinesVentaDTO[] linesVenta;

	private TestCliente cliente;

	private String valorTotalVenta;

	private Date fechaVenta;
}

File: Front/src/app/shared/models/VentaDTO.ts
import { DetalleVentaDTO } from "./DetalleVentaDTO";
import { ClienteDTO } from './ClienteDTO';

export class VentaDTO {
	public cliente: ClienteDTO;
	public fechaVenta: Date = new Date();
	public valorTotalVenta: number;
	public linesVenta: Array<DetalleVentaDTO>;

}
